src/main/app/case/answers/getAnswerRows.ts

```diff
@@ -12,10 +12,12 @@ export const getAnswerRows = function (section: Sections): GovUkNunjucksSummary[
     language,
     isDivorce,
     formState,
+    userEmail,
   }: {
     language: 'en' | 'cy';
     isDivorce: boolean;
     partner: string;
+    userEmail?: string;
```

Comment:
does it have to be optional?

Reply:
👍 Good spot, I've made this required in e21275c.